src/styles/programsList.css

```
@@ -0,0 +1,67 @@
+
+  .card-group.programsList{
+    margin-left:1rem;
+  }
+  .programsList-card.card {
+    min-width: 23%;
+    background-color: #f4f4f4;
+    padding: 5px;
+    margin-bottom:1rem;
+    margin-right:1rem;
+
+  }
+  .stretched-link:after {
+    content: "";
+  }
+  
+  .programsList-card h2 {
+    color: black;
+    padding-top: 0px;
+    margin-top: 15px;
+    font-size: 2rem;
+  }
+  
+  .programsList-card.card:nth-of-type(12n+1),
+  .programsList-card.card:nth-of-type(12n+4),
+  .programsList-card.card:nth-of-type(12n+7),
+  .programsList-card.card:nth-of-type(12n+10) {
+    border-left: 1rem solid var(--uog-red);
+  }
+
+  .programsList-card.card:nth-of-type(12n+2),
+  .programsList-card.card:nth-of-type(12n+5),
+  .programsList-card.card:nth-of-type(12n+8),
+  .programsList-card.card:nth-of-type(12n+11) {
+    border-left: 1rem solid var(--uog-yellow);
+  }
+  .programsList-card.card:nth-of-type(12n+3),
+  .programsList-card.card:nth-of-type(12n+6),
+  .programsList-card.card:nth-of-type(12n+9),
+  .programsList-card.card:nth-of-type(12n+12) {
+    border-left: 1rem solid var(--uog-blue);
+}
+  
+  .programsList-card:hover {
```

```suggestion
  .programsList-card:hover,
  .programsList-card:focus {
```

So keyboard users also see the grey background when they tab to a program card.